# Request: An array of strings
string start
string[] destinations

---
# Response: An array of strings
string[] GA_result